tests/regression/56-witness/01-base-lor-enums.c

```diff
@@ -0,0 +1,33 @@
+// PARAM: --enable ana.int.enums --set witness.yaml.validate 01-base-lor-enums.yml
+
+int main() {
+  int r; // rand
+  int x;
+  switch (r) {
+    case 0:
+      x = 1;
+      break;
+    case 1:
+      x = 3;
+      break;
+    default:
+      x = 6;
+      break;
+  }
+  ; // SUCCESS (witness)
+  ; // UNKNOWN (witness)
```

Comment:
The testing script doesn't care about the exclamation mark at all, so it works here as well. And you're right that it makes sense to use `UNKNOWN!` for these, especially since both also contain an invariant that holds, but the given `LOr` handling cannot prove. So I made these changes.